src/index.html

```diff
@@ -13,10 +13,96 @@
     <title>Moyo header</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        alt="logo"
+      />
```

Comment:
Fix it 
<img width="515" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/abb6e54e-6ffe-40f4-a489-4082af08e4bc">